lib/publiccloud/provider.pm

```diff
@@ -313,6 +313,9 @@ sub create_instances {
             # Install server's ssh publicckeys to prevent authenticity interactions
             assert_script_run(sprintf('ssh-keyscan %s >> ~/.ssh/known_hosts', $instance->public_ip));
         }
+        # Performance data: boottime
+        my $btime = $instance->measure_boottime($instance);
```

Comment:
this calls should be controlled by some variable so we can switch this off if it will start bringing troubles ...

Reply:
the enabling variables checks are inside those functions `measure` and `store`.
But, do you mean that those function calls shall be moved outside `create_instance` instead?

Reply:
As discussed, the enabling variables checks location could be ok.